packages/api/src/post-build.js

```diff
@@ -167,6 +167,50 @@ class PostBuildHandler {
     return result;
   }
 
+  static cleanBuildInfo (metadata) {
+    const timestampNumb = Date.parse(metadata.timestamp);
+    const timestamp = isNaN(timestampNumb) ? new Date() : new Date(timestampNumb);
+
+    const returnVal = {
+      repoId: firebaseEncode(decodeURIComponent(metadata.repoId)),
+      organization: metadata.organization,
```

Comment:
what if I'm malicious and create an organization that's like `foo/johnny-drop-tables`. It seems like we might want to encode all fields for firebase, regardless of whether we think they could contain a `/`.

Reply:
I only escaped strings that will be used as keys in Firestore. the organization value will always be treated as a string. The only operation that it goes through are 
- a firestore update with organization: 'user input"
- a firestore read 
- a res.send of the json object read